cypress/support/fragments/checkout/confirmMultipieceCheckOutModal.js

```diff
@@ -0,0 +1,39 @@
+import { Button, Modal, MultiColumnListRow, KeyValue, including, HTML } from '../../../../interactors';
+
+const confirmModal = Modal('Confirm multipiece check out');
+const checkOutButton = confirmModal.find(Button('Check out'));
+const cancelButton = confirmModal.find(Button('Cancel'));
+const numberOfPieces = confirmModal.find(KeyValue('Number of pieces'));
+const descriptionOfPieces = confirmModal.find(KeyValue('Description of pieces'));
+const numberOfMissingPiecesKeyValue = confirmModal.find(KeyValue('Number of missing pieces'));
+const descriptionOfMissingPiecesKeyValue = confirmModal.find(KeyValue('Description of missing pieces'));
+
+export default {
+  confirmMultipleCheckOut:() => {
+    cy.do(checkOutButton.click());
+    cy.expect(MultiColumnListRow().exists());
```

Comment:
make expectation more special. Add root element or link to special test data. Or replace to absent of modal window

Reply:
Done